docs/01-app/02-guides/instrumentation.mdx

```diff
@@ -1,5 +1,6 @@
 ---
-title: Instrumentation
+title: How to set up instrumentation
```

Comment:
```suggestion
title: How to set up server instrumentation
```

I was thinking this, but if this is truly a guide, maybe it should have both server _and_ client instrumentation and then we can link out to the more specific API reference pages

Reply:
Can be follow up!